client/my-sites/backup/main.jsx

```diff
@@ -51,7 +51,9 @@ const BackupPage = ( { queryDate } ) => {
 
 	const moment = useLocalizedMoment();
 	const parsedQueryDate = queryDate ? moment( queryDate, INDEX_FORMAT ) : moment();
-	const selectedDate = useDateWithOffset( parsedQueryDate );
+	const selectedDate = useDateWithOffset( parsedQueryDate, {
+		keepLocalTime: true,
```

Comment:
This is the only place in which we want to maintain the local time because here is where we parse the date from the URL.